model_compression_toolkit/keras/keras_implementation.py

```diff
@@ -277,12 +277,10 @@ def gptq_training(self,
         Returns:
             Updated graph after GPTQ.
         """
-
-        return gptq_training_wrapper(graph_float,
-                                     graph_quant,
-                                     representative_data_gen,
-                                     gptq_config,
-                                     fw_info)
+        gptq_trainer = KerasGPTQTrainer(graph_float, graph_quant, gptq_config, fw_info)
```

Comment:
Keras Implementation should only return the trainer and the rest should be in common no?

Reply:
why not put this function in common (FrameworkImplementation), and set the trainer in the init?
something like self.gptq_trainer = KerasGPTQTrainer

Reply:
In the init of what?

Reply:
@elad-c  I think it is not the correct way to implement this
We should have a common function that gets a GPTQTrainer and returns a graph.

Reply:
Ok this sound good.